extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationNotifications.js

```diff
@@ -0,0 +1,375 @@
+require(
+	['jquery', 'wikia.window', 'wikia.nirvana'],
+	function($, window, nirvana) {
+		'use strict';
+
+		var WallNotifications = {
+			init: function () {
+				this.bucky = window.Bucky('WallNotifications');
+				this.updateInProgress = false; // we only want 1 update simultaneously
+				this.notificationsCache = {}; // HTML for "trays" for different Wiki ids
+				this.wikiShown = {}; // all open "trays" (Wiki Notifications) - list of Wiki ids
+				this.fetchedCurrent = false; // we only want to force-fetch notifications for current Wiki once
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ATrailingComment)